Userland/Applications/HexEditor/HexEditorWidget.cpp

```diff
@@ -356,44 +356,43 @@ void HexEditorWidget::update_inspector_values(size_t position)
         value_inspector_model->set_parsed_value(ValueInspectorModel::ValueType::Double, "");
     }
 
-    // FIXME: This probably doesn't honour endianness correctly.
```

Comment:
How is it fixed now?

Reply:
Oh good call. I believe I thought that was an erroneous FIXME, but now I'm not as sure. I'll look at it again if I decide to remove it

Reply:
Yeah confirmed. UTF-8 is byte aligned, so there is no difference between little endian and big endian